### PR TITLE
[DOC] Wrong English & Render

### PR DESCRIPTION
**Pull request by GalyaZalesskaya**
What was done:
- Checked grammar, punctuation and rendering for
- - Main page
- - Introduction
- - Tutorials
- - Explanation

- Checked links
- Added more consistency and similarity among pages
- Hidden the `Coming soon` blocks together with tables that contained only `N/A` values in the Explanation, so it will not be shown but will stay in the source
- Updated tutorial with `otx explain` with more details and image

### CODE REVIEW

**Review by chuneuny-emily — DISMISSED** [review]

File: docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst [hunk 1]
```diff
@@ -151,27 +154,27 @@ Each subset dataset has 8, 16, and 24 images, respectively.
 
 Unlike other tasks, two things are considered to use self-supervised learning:
 
-- `--train-data-root` must be set to a directory only containing images, not ground truths.
-  DetCon uses pseudo masks created in `detcon_mask` directory for training. If they are not created yet, they will be created first.
-- `--val-data-root` is not needed.
+- ``--train-data-root`` must be set to a directory only containing images, not ground truths.
```
Comment (eunwoosh) [inline docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst]:
```suggestion
- ``--train-data-roots`` must be set to a directory only containing images, not ground truths.
```

File: docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst [hunk 2]
```diff
@@ -151,27 +154,27 @@ Each subset dataset has 8, 16, and 24 images, respectively.
...
+  DetCon uses pseudo masks created in ``detcon_mask`` directory for training. If they are not created yet, they will be created first.
+- ``--val-data-root`` is not needed.
```
Comment (eunwoosh) [inline docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst]:
```suggestion
- ``--val-data-roots`` is not needed.
```

File: docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst [hunk 3]
```diff
@@ -151,27 +154,27 @@ Each subset dataset has 8, 16, and 24 images, respectively.
...
 
 To enable self-supervised training, the command below can be executed:
 
 .. code-block::
 
   $ otx train otx/algorithms/segmentation/configs/ocr_lite_hrnet_s_mod2/template.yaml \
-      --train-data-root=tests/assets/common_semantic_segmentation_dataset/train/images \
-      params \
-      --algo_backend.train_type=SELFSUPERVISED
+              --train-data-root=tests/assets/common_semantic_segmentation_dataset/train/images \
```
Comment (eunwoosh) [inline docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst]:
```suggestion
              --train-data-roots=tests/assets/common_semantic_segmentation_dataset/train/images \
```

File: docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst [hunk 4]
```diff
@@ -151,27 +154,27 @@ Each subset dataset has 8, 16, and 24 images, respectively.
...
 
 After self-supervised training, pretrained weights can be use for supervised (incremental) learning like the below command:
 
 .. code-block::
 
   $ otx train otx/algorithms/segmentation/configs/ocr_lite_hrnet_s_mod2/template.yaml \
-      --train-data-root=tests/assets/common_semantic_segmentation_dataset/train \
-      --val-data-roots=tests/assets/common_semantic_segmentation_dataset/val \
-      --load-weights={PATH/PRETRAINED/WEIGHTS}
+              --train-data-root=tests/assets/common_semantic_segmentation_dataset/train \
```
Comment (eunwoosh) [inline docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst]:
```suggestion
              --train-data-roots=tests/assets/common_semantic_segmentation_dataset/train \
```

File: docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst [hunk 5]
```diff
@@ -203,16 +207,16 @@ It can be launched only with supervised (incremental) training type.
 .. code-block::
 
   $ otx train otx/algorithms/segmentation/configs/ocr_lite_hrnet_s_mod2/template.yaml \
-      --train-data-root=tests/assets/common_semantic_segmentation_dataset/train \
-      --val-data-roots=tests/assets/common_semantic_segmentation_dataset/val \
-      params \
-      --learning_parameters.enable_supcon=True
+              --train-data-root=tests/assets/common_semantic_segmentation_dataset/train \
```
Comment (eunwoosh) [inline docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst]:
```suggestion
              --train-data-roots=tests/assets/common_semantic_segmentation_dataset/train \
```

**Review by sungmanc — DISMISSED** [review]
Thanks for the detailed checks

**Review by supersoob — DISMISSED** [review]
Thank you Galina !! :)

File: docs/source/guide/tutorials/advanced/api_tutorial.rst [hunk 6]
```diff
@@ -86,10 +86,10 @@ The next step is to set up a dataset:
 .. code-block::
 
     dataset_adapter = get_dataset_adapter(task_type = model_template.task_type,
-                                            # set a path to the root folder of the wgisd repository
-                                            train_data_roots="./wgisd",
-                                            val_data_roots="./wgisd",
-                                            test_data_roots="./wgisd")
+                                          # set a path to the root folder of the wgisd repository
```
Comment (eunwoosh) [inline docs/source/guide/tutorials/advanced/api_tutorial.rst:89]: As far as I know, no space when passing arguments is a convention.

File: docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst [hunk 7]
```diff
@@ -39,7 +52,7 @@ Download and prepare `VOC2012 dataset <http://host.robots.ox.ac.uk/pascal/VOC/vo
 .. image:: ../../../../../utils/images/voc_example.png
   :width: 600
 
-The dataset contains a set of RGB images with 20 semantic labels such as aeroplane, bicycle, bird, car, person, etc. The images are stored in the following format:
+The dataset contains a set of RGB images with 20 semantic labels such as airplane, bicycle, bird, car, person, etc. The images are stored in the following format:
```
Comment (eunwoosh) [inline docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst]:
```suggestion
The dataset contains a set of RGB images with 20 semantic labels such as aeroplane, bicycle, bird, car, person, etc. The images are stored in the following format:
```
As far as I know, `aeroplane` is used in official VOC dataset homepage.

File: docs/source/guide/tutorials/advanced/api_tutorial.rst [hunk 8]
```diff
@@ -86,10 +86,10 @@ The next step is to set up a dataset:
...
+                                          train_data_roots = "./wgisd",
+                                          val_data_roots = "./wgisd",
+                                          test_data_roots = "./wgisd")
```
Comment (goodsong81) [inline docs/source/guide/tutorials/advanced/api_tutorial.rst]:
```suggestion
                                          train_data_roots="./wgisd",
                                          val_data_roots="./wgisd",
                                          test_data_roots="./wgisd")
```

File: docs/source/guide/tutorials/advanced/api_tutorial.rst [hunk 9]
```diff
@@ -102,10 +102,10 @@ Set up the environment and the task
...
 
     environment = TaskEnvironment(
-        model=None,
-        hyper_parameters=hyper_parameters,
-        label_schema=labels_schema,
-        model_template=model_template)
+        model = None,
+        hyper_parameters = hyper_parameters,
+        label_schema = labels_schema,
+        model_template = model_template)
```
Comment (goodsong81) [inline docs/source/guide/tutorials/advanced/api_tutorial.rst]:
```suggestion
        model=None,
        hyper_parameters=hyper_parameters,
        label_schema=labels_schema,
        model_template=model_template)
```

**Review by eunwoosh — COMMENTED** [review]
Thanks for your work! I think document get much better. I left comment about some tinies. please take a look.

File: docs/source/guide/tutorials/advanced/api_tutorial.rst [hunk 10]
```diff
@@ -135,9 +135,9 @@ To perform validation we need to infer our model on the validation dataset, crea
         InferenceParameters(is_evaluation=True))
 
     resultset = ResultSetEntity(
-        model=output_model,
-        ground_truth_dataset=validation_dataset,
-        prediction_dataset=predicted_validation_dataset,
+        model = output_model,
+        ground_truth_dataset = validation_dataset,
+        prediction_dataset = predicted_validation_dataset,
```
Comment (goodsong81) [inline docs/source/guide/tutorials/advanced/api_tutorial.rst]:
```suggestion
        model=output_model,
        ground_truth_dataset=validation_dataset,
        prediction_dataset=predicted_validation_dataset,
```

File: docs/source/guide/tutorials/advanced/api_tutorial.rst [hunk 11]
```diff
@@ -168,9 +168,9 @@ To validate the OpenVINO™ IR model, we need to create an openvino task first a
...
```
Comment (goodsong81) [inline docs/source/guide/tutorials/advanced/api_tutorial.rst]:
```suggestion
        model=output_model,
        ground_truth_dataset=validation_dataset,
        prediction_dataset=predicted_validation_dataset,
```

File: docs/source/guide/tutorials/advanced/api_tutorial.rst [hunk 12]
```diff
@@ -229,9 +229,9 @@ You can validate the optimized model as the usual model, for example for the NNC
 
     # ResultSetEntity creating with optimized_nncf_model
     resultset = ResultSetEntity(
-        model=optimized_nncf_model,
-        ground_truth_dataset=validation_dataset,
-        prediction_dataset=predicted_validation_dataset,
+        model = optimized_nncf_model,
+        ground_truth_dataset = validation_dataset,
+        prediction_dataset = predicted_validation_dataset,
```
Comment (goodsong81) [inline docs/source/guide/tutorials/advanced/api_tutorial.rst]:
```suggestion
        model=optimized_nncf_model,
        ground_truth_dataset=validation_dataset,
        prediction_dataset=predicted_validation_dataset,
```

File: docs/source/guide/tutorials/advanced/api_tutorial.rst [hunk 13]
```diff
@@ -254,10 +254,10 @@ Let's assume, that we have already trained the model and we want to use it in ou
...
     # create new environment
     environment = TaskEnvironment(
-            model=None,
-            hyper_parameters=hyper_parameters,
-            label_schema=read_label_schema(weights_path),
-            model_template=template,
+            model = None,
+            hyper_parameters = hyper_parameters,
+            label_schema = read_label_schema(weights_path),
+            model_template = template,
```
Comment (goodsong81) [inline docs/source/guide/tutorials/advanced/api_tutorial.rst]:
```suggestion
            model=None,
            hyper_parameters=hyper_parameters,
            label_schema=read_label_schema(weights_path),
            model_template=template,
```

File: docs/source/guide/tutorials/advanced/api_tutorial.rst [hunk 14]
```diff
@@ -281,8 +281,8 @@ We can convert these steps to function and use it in a loop with multiple images
         empty_annotation = AnnotationSceneEntity(annotations=[], kind=AnnotationSceneKind.PREDICTION)
 
         item = DatasetItemEntity(
-            media=Image(cv2.cvtColor(frame, cv2.COLOR_BGR2RGB)),
-            annotation_scene=empty_annotation,
+            media = Image(cv2.cvtColor(frame, cv2.COLOR_BGR2RGB)),
+            annotation_scene = empty_annotation,
```
Comment (goodsong81) [inline docs/source/guide/tutorials/advanced/api_tutorial.rst]:
```suggestion
            media=Image(cv2.cvtColor(frame, cv2.COLOR_BGR2RGB)),
            annotation_scene=empty_annotation,
```

**Review by goodsong81 — DISMISSED** [review]
What a great work!
I'll apply some needed changes in direct commits.
Thank you for the effort :)

**Review by goodsong81 — APPROVED** [review]

**Review by eunwoosh — APPROVED** [review]

**Pull request merged**